sixMans/game.py

```diff
@@ -412,17 +431,17 @@ async def process_team_select_vote(self, emoji, member, added=True):
                 runner_up = num_votes
 
             total_votes += num_votes
-        
+
         pending_votes = len(self.players) - total_votes
 
+
         # Vote Complete if...
         if added and (pending_votes + runner_up) <= running_vote[1]:
             # action and update first - help with race conditions
-            if self.teamSelection.lower() == Strings.VOTE_TS.lower():
-                self.teamSelection = SELECTION_MODES[running_vote[0]]
-                embed = self._get_vote_embed(vote=votes, winning_vote=running_vote[0])
-                await self.info_message.edit(embed=embed)
-                await self.process_team_selection_method()
+            self.teamSelection = SELECTION_MODES[running_vote[0]]
```

Comment:
I'm guessing this code works because the vote is sorted by team selection with most votes? Just looking for confirmation.

Does this change how the team selection embed works visually? Hoping not, but again, looking for confirmation.